synapse/models/infotech.py

```diff
@@ -1366,6 +1372,34 @@ def getModelDefs(self):
                     }),
                 )),
 
+                ('it:dev:label', {}, (
```

Comment:
Ok, i finally grok it :D 

`it:dev:repo:label` -> `it:dev:repo:issue:label`
`it:dev:label` -> `it:dev:repo:label`

final answer ;)